edgelet/build/linux/debian/arm32v7/package-libiothsm.sh

```diff
@@ -42,7 +42,7 @@ run_command "cd /target/hsm/build && make package"
 
 # Old CPACK produces non-standard deb package filenames.
 # This renames them
-for f in $BUILD_DIR/$PACKAGE_NAME-*-Linux.deb ; do
+for f in $BUILD_DIR/$PACKAGE_NAME-*-Linux-libraries.deb ; do
```

Comment:
Out of curiosity, was this renamed this way for a reason, if so why in this way?

Reply:
CPack appends the component name to the filename, and `libraries` is the component name I set.